topic-operator/src/main/java/io/strimzi/operator/topic/TopicOperatorUtil.java

```diff
@@ -39,19 +37,6 @@ public class TopicOperatorUtil {
 
     private TopicOperatorUtil() { }
 
-    /**
-     * Create a new Kubernetes client instance.
-     *
-     * @param id Caller id.
-     * @return Kubernetes client.
-     */
-    public static KubernetesClient createKubernetesClient(String id) {
-        return new OperatorKubernetesClientBuilder(
-            "strimzi-topic-operator-" + id,
-            TopicOperatorMain.class.getPackage().getImplementationVersion())
-            .build();
-    }
-
```

Comment:
I don't think it adds any value. So it is just taking memory.

Reply:
As I said, same client configuration everywhere. Right now we are only setting a naming convention, but there could be more stuff in the future.

Reply:
If there is something added to the configuration what actually needs to be used in the tests, we can add it there together with all the other tests we have that follow exactly the same pattern.

Reply:
Ok, if we want to be consistent with the rest of the project, then we should do the same with createKafkaAdminClient and createCruiseControlClient. It would be good to also refactor the CruiseControlClient.create to take a config object like for the other clients.

Reply:
I don't think that uses Kubernetes client? So it should be probably done in a separate PR?

Reply:
Fine. Can you please raise an improvement?